### ISSUE TITLE
primary+space doesn't work in a new separate window

### ISSUE THREAD

**Issue by sojusnik** [zim-desktop-wiki/zim-desktop-wiki#1593]
While primary+space does work in the main window of Zim to switch between the text are and the sidebar, it's not working in a separate window.

Observed with `0.74.0-SNAPSHOT-t202109171459~ubuntu21.04.1`.

**Comment by jaap-karssenberg** [comment]
Yes, this was never implemented for the secondary window. Missing feature,
not a bug.

Op za 18 sep. 2021 11:40 schreef sojusnik ***@***.***>:
